bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -48,7 +48,7 @@ <h2 class="c-sub-navigation-title">{{ _('Enterprise') }}</h2>
     </p>
     <div class="cta-container">
       <div class="mzp-c-button-download-container">
-        <a id="primary-download-button" href="#download" class="mzp-c-button mzp-t-product">{{ ftl('download-button-download') }}</a>
+        <a id="primary-download-button" href="#download" class="mzp-c-button mzp-t-product mzp-t-xl">{{ ftl('download-button-download') }}</a>
```

Comment:
Curious why we add `mxp-t-xl` here, but pages like `/firefox/new/` still have the default (smaller) button sizes? Is there a new rule that should be followed here?

![image](https://user-images.githubusercontent.com/400117/86476872-436c9400-bd3f-11ea-9c3e-bb31e2fa6821.png)


![image](https://user-images.githubusercontent.com/400117/86476834-2df76a00-bd3f-11ea-8045-dfac1227aaa6.png)

Reply:
No rule, I was just going with my gut.

Reply:
Ok, well if there's one button where I think people might freak out a little if we made it smaller, it's probably the one on /new. Worth making it match the size you've applied to /enterprise?